scripts/node/README.md

```diff
@@ -0,0 +1,37 @@
+# Steps to benchmark node scanning 
+
+## Pre steps:
+
+1.Make sure docker is installed on your local laptop
```

Comment:
```suggestion
1. Make sure docker is installed on your local laptop
```
Thank you for converting to Markdown! Great to see the renderer picking it up.
Looking at [the rendered file](https://github.com/stackrox/scanner/blob/01bfda2267e65abe911dd949b49d0c93022279b0/scripts/node/README.md), it seems that the renderer didn't pick up the ordered list.
I suspect this is because there are missing whitespaces between the number and the text ([see also: Spec](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#lists)). See my change for an example. 

I recommend this to be done for all list items in this file.
Maybe we should also capitalize the first word of each list item.